<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>451</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Sitka</family>
         <pointsize>18</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>WELCOME TO STUDENT DATABSE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poor Richard</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Your Role:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Admin_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADMIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Student_name">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STUDENT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitMain_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>671</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 4px solid rgb(223,212,122)
 qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>Admin_button</zorder>
   <zorder>Student_name</zorder>
   <zorder>ExitMain_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Admin_button</sender>
   <signal>clicked()</signal>
   <receiver>Admin_button</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Student_name</sender>
   <signal>clicked()</signal>
   <receiver>Student_name</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
